docs/rules/no-unsafe-finally.md

```diff
@@ -0,0 +1,99 @@
+# Disable Unsafe Statements in `finally` Block (no-unsafe-finally)
+
+Using control flow operations in a `finally` block may lead to unexpected behavior. For example:
+
+```js
+try {
+   return 1;
+} catch(err) {
+   return 2;
+} finally {
+   return 3;
+}
+// > 3
+
+try {
+    throw new Error("Try");
+    return 1;
+} catch(err) {
+    throw err;
+    return 2;
+} finally {
+    throw new Error("Finally");
+    return 3;
+}
+
+// > Uncaught Error: Finally(...)
+```
+
+## Rule Details
+
+This rule forbids the usage of `return`, `throw`, `break` and `continue` operations inside a `finally` block, ignoring function declarations.
```

Comment:
s/operations/statements/
s/a `finally` block/`finally` blocks/

Is function definitions a clear way to mean either of the following?
* function declarations: function foo () { }
* function expressions: var foo = function () { }